browsers/chrome.json

```diff
@@ -539,19 +539,26 @@
         "77": {
           "release_date": "2019-09-10",
           "release_notes": "https://chromereleases.googleblog.com/2019/09/stable-channel-update-for-desktop.html",
-          "status": "current",
+          "status": "retired",
           "engine": "Blink",
           "engine_version": "77"
         },
         "78": {
-          "status": "beta",
+          "release_date": "2019-10-22",
+          "release_notes": "https://chromereleases.googleblog.com/2019/10/stable-channel-update-for-desktop_22.html",
+          "status": "retired",
```

Comment:
This should be 'current'.

Reply:
Whoops, don’t know how that happened — lemme fix that up!

Reply:
```suggestion
          "status": "current",
```